specification/communication/data-plane/SipRouting/preview/2024-11-15-preview/communicationservicessiprouting.json

```diff
@@ -365,6 +376,18 @@
         "enabled": {
           "description": "Enabled flag",
           "$ref": "#/definitions/Enabled"
+        },
+        "stepOutOfSignaling": {
```

Comment:
Do API user understand what "Signaling" means and "stepping out of it"? Are these terms described anywhere in industry standards or in ACS docs?

Reply:
Good question, Signaling is and industry standard naming to describe the control part of the telephony communication (establish, tear down, transfer call).
StepOutOfSignaling is how we call the feature, but as it's not release to public yet, we don't have it in the ACS docs so far. But the property deserves better description.